### PR TITLE
Multiple match support 

### PR DESCRIPTION
**Pull request by Fueredoriku**
- Main menu is interactable after match completion
- MatchManager properly reset itself when match is completed
- Starting items are now set in staticInfo
- Game supports having just a body and a barrel again.

### CODE REVIEW

**Review by toberge — CHANGES_REQUESTED** [review]
Looks like a fix, though I left some questions for you, and there's a nitpick you've got to fix :triumph: 

Also, when the game starts anew, all players who already joined are still there. I suppose this makes sense, though as far as I know there's no way for players to leave?

File: Assets/Scripts/Gamestate/MatchController.cs [hunk 1]
```diff
@@ -79,6 +78,10 @@ void Start()
 
         #endregion Singleton boilerplate
 
+        if (rounds.Count == 0)
+        {
+            PlayerInputManagerController.Singleton.playerInputs.ForEach(input => input.GetComponent<PlayerIdentity>().resetItems());
```
[the remark this answers is not in the document]
Comment (Fueredoriku) [reply Assets/Scripts/Gamestate/MatchController.cs:83]: Yep

File: Assets/Scripts/UI/MainMenuController.cs [hunk 2]
```diff
@@ -47,6 +48,10 @@ void Start()
             //playerInputManagerController.GetComponent<MultiplayerEventSystem>().playerRoot = defaultMenu;
             TransferExistingInputs();
         }
+        else
```
[the remark this answers is not in the document]
Comment (Fueredoriku) [reply Assets/Scripts/UI/MainMenuController.cs:51]: Nope, good point

File: Assets/Scripts/Augment/GunFactory.cs [hunk 3]
```diff
@@ -24,15 +24,15 @@ public static GunStats GetGunStats(Item body, Item barrel, Item extension)
...
+        extension?.augment.GetComponent<GunExtension>().BuildStats(stats);
         return stats;
     }
 
     public static string GetGunName(Item body, Item barrel, Item extension)
     {
         OverrideName result = StaticInfo.Singleton.SecretNames.Where(x => (x.Body == body && x.Barrel == barrel && x.Extension == extension)).FirstOrDefault();
         if (!(result.Name is null)) { return result.Name; }
-        return $"The {body.secretName} {extension.secretName} {barrel.secretName}";
+        return $"The {body.secretName} {extension?.secretName} {barrel.secretName}";
```
[the remark this answers is not in the document]
Comment (Fueredoriku) [reply Assets/Scripts/Augment/GunFactory.cs:35]: Pain

**Comment by Fueredoriku** [comment]
> though as far as I know there's no way for players to leave?

Nope, I'll leave that for #90

**Review by toberge — APPROVED** [review]
You fixed the issues I pointed out, now's the time to give a :+1:

**Pull request merged**